core/src/main/scala/fs2/netty/Network.scala

```diff
@@ -174,6 +175,7 @@ object Network {
     }
   }
 
+  //TODO: Why not use the Netty methods 
```

Comment:
For example, 
```
import io.netty.channel.epoll.Epoll
import io.netty.channel.kqueue.KQueue
if (Epoll.isAvailable) {epoll elg} else if (KQueue.isAvailable) {kqueue elg} else {nio elg}

// Don't see any the incubator package in current Netty version, but presumably it would look similar to above
```

[Epoll source code](https://github.com/netty/netty/blob/4.1/transport-native-epoll/src/main/java/io/netty/channel/epoll/Epoll.java) for more details.

Reply:
So. Much. Nicer. Let's do this.